my-code/movie-search/src/Containers/MovieDetails/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
```

Comment:
As you're not dispatching several actions on your async calls, if we open a movie detail, go back and open another movie, we still see the old details for a second before it changes

Reply:
Yes, I noticed this too. Your earlier recommendation on async catches this eventuality?